src/services/entities/transformers/getLongTriggerCode.test.ts

```diff
@@ -0,0 +1,33 @@
+import getLongTriggerCode from "./getLongTriggerCode"
+
+describe("getLongTriggerCode", () => {
+  it("Should return the short version of the trigger code", () => {
+    const result = getLongTriggerCode("PR01")
+
+    expect(result).toBe("TRPR0001")
+  })
```

Comment:
I'd like to see how it handles with `TRPS0008`

Reply:
Cant think of a reason why it would have problems but I just ran it through a quick test to be sure. Handles it fine:

![Screenshot 2024-07-26 at 11 39 59](https://github.com/user-attachments/assets/3aad5a97-accc-4704-b7c0-bdc3b350dd1a)

Reply:
Can add a test case to confirm it handles `PS` as well as `PR` codes